tests/unit/test__download.py

```diff
@@ -23,8 +23,7 @@
 
 
 EXAMPLE_URL = (
-    "https://www.googleapis.com/download/storage/v1/b/"
-    "{BUCKET}/o/{OBJECT}?alt=media"
+    "https://www.googleapis.com/download/storage/v1/b/" "{BUCKET}/o/{OBJECT}?alt=media"
```

Comment:
(nit) We can merge this into a single string literal.

Reply:
3f6d81d